### ISSUE TITLE
Ntfy support

### ISSUE THREAD

**Issue by JohnFawkes** [njic/medassist#16]
I plan on installing this when i get some free time but seeing ntfy support would be really useful for discord notifications or phone notifications instead of email.

Thanks

**Comment by John710** [comment]
It would be good to add [Shoutrrr](https://containrrr.dev/shoutrrr/v0.8/) support so that you can configure notifications in any convenient service.

**Comment by njic** [comment]
I knew about Ntfy and Gotify, but Shoutrrr is a new one for me. I'm definitely considering adding support for notifications and @John710 comment sounds like Shoutrrr support might work with any notification platform. Am I right?

I don't mind adding any feature that is useful, simple and foss

**Comment by John710** [comment]
> Am I right?

I'm not a developer, but many of the services I use, work with notifications through Shoutrrr.

**Comment by phampyk** [comment]
Yes please to this one!! I just found this project and I'm gonna give it a go, but it would be so much better being able to choose where you want to be notified.

Apprise and Shoutrrr are the main ones I know, whatever seems easier to implement.
Thank you so much for this tho!

**Comment by NeoXTof** [comment]
or more generally, add support for notifications using webhooks. Ntfy, Gotify, and many others can be used with simple webhook support

**Comment by njic** [comment]
Thanks, this might be helpful since my understanding is not there yet. I'll research more and probably start working on this ASAP

**Comment by hairyfred** [comment]
Any sort of web hook support would be good (would love to hook this into home assistant for notifications/automations for example)

**Comment by njic** [comment]
yeah, webhook incoming soon!
